src/api.js

```diff
@@ -1274,16 +1237,13 @@ api.declare({
   route:      '/poll-task-url/:provisionerId/:workerType',
   name:       'pollTaskUrls',
   stability:  API.stability.stable,
-  scopes: [
-    [
-      // Legacy scopes
+  scopes: {AnyOf: [
+    'queue:poll-task-urls:<provisionerId>/<workerType>',
```

Comment:
for consistency in docs, I suggest prefixing variables with `task.` even if that means we have to add `task: {...}` when we do params...

Reply:
hmm, maybe not, because there is a lot of places where this makes less sense...


maybe we should do it the other way around and get rid of `task.` ?

Reply:
It's not been qualified in the docs so far, without causing a lot of confusion, so I think that we should omit it.